src/components/common/VideoCardMenu.vue

```diff
@@ -116,7 +115,7 @@
       {{ $t("component.videoCard.uploadScript") }}
     </v-list-item>
     <v-list-item
-      v-if="video.type !== 'placeholder'"
+      v-if="isChattable"
```

Comment:
I was gonna suggest changing `isLive` to support `upcoming` because every usage of it in this file also checks if it's upcoming.... but this works too

Reply:
The difference is additional check for `type` (maybe `type === 'stream'` is too strict... forgot to check if premieres fall into this category) but `isLive` indeed can be modified separately if it's never needed w/o upcoming.